internal/cli/universal_login_customize.go

```diff
@@ -18,9 +46,195 @@ func customizeUniversalLoginCmd(_ *cli) *cobra.Command {
 		Example: `  auth0 universal-login customize
   auth0 ul customize`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			return fmt.Errorf("not yet implemented")
+			var universalLoginBrandingData *universalLoginBrandingData
+
+			if err := ansi.Spinner("Fetching Universal Login branding data", func() (err error) {
+				universalLoginBrandingData, err = fetchUniversalLoginBrandingData(cmd.Context(), cli.api, cli.tenant)
+				return err
+			}); err != nil {
+				return err
+			}
+
+			cli.renderer.JSONResult(universalLoginBrandingData)
+
+			return nil
 		},
 	}
 
 	return cmd
 }
+
+func fetchUniversalLoginBrandingData(
+	ctx context.Context,
+	api *auth0.API,
+	tenantDomain string,
+) (*universalLoginBrandingData, error) {
+	group, ctx := errgroup.WithContext(ctx)
+
+	group.Go(func() (err error) {
+		return ensureCustomDomainIsEnabled(ctx, api)
+	})
```

Comment:
Is this meant to be in parallel with the other requests? I could also see an argument for this check happening before the normal batch of requests go out. A key advantage I see is that there will only be one error returned for this specific case rather than potentially multiple.

Reply:
Improved in https://github.com/auth0/auth0-cli/pull/861/commits/dabdece59de62081aaa282a1731e6447b12f6571